packages/docs/src/content/docs/overview/configuration.md

```diff
@@ -46,9 +46,9 @@ If your project structure does not match the default `entry` and `project`
 files, you can customize them. Here's an example configuration to include `.js`
 files in the `scripts` folder:
 
-```json title="knip.json"
+```json title="knip.jsonc"
 {
-  "$schema": "https://unpkg.com/knip@3/schema.json",
+  "$schema": "https://unpkg.com/knip@3/schema-jsonc.json",
```

Comment:
Let's keep it simple for people new here. Other options JSONC and TypeScript are offered later on. So I think it's better not to introduce JSONC here.